pkg/collector/corechecks/containers/docker.go

```diff
@@ -217,6 +217,10 @@ func (d *DockerCheck) Run() error {
 				}
 			}
 
+			if c.KernMemUsage >= 0 && c.SoftMemLimit < uint64(math.Pow(2, 60)) {
```

Comment:
Why do we need the check on SoftMemLimit here? And can KernMemUsage be < 0? If not let's remove the checks entirely.

Reply:
Copy/Pasta fail. I'll fix that. It shouldn't ever be < 0, so I'll drop the check.

Reply:
I've fixed this and dropped the check.